### PR TITLE
added bootstrap and friendly id

### PR DESCRIPTION
**Pull request by joney000**
added bootstrap design and friendly id feature.

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1,2 @@
+finish
```
Comment (alienware) [inline .byebug_history:1]: Remove this file from VC.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
 //= require jquery_ujs
+//= require twitter/bootstrap
 //= require turbolinks
-//= require_tree .
+//= require_tree .
```
Comment (alienware) [inline app/assets/javascripts/application.js]: New line

File: app/views/articles/_form.html.haml [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
-    = f.label :text
-    %br
-      = f.text_area :text
-  %p
-    = f.submit
\ No newline at end of file
+  
+  %br
+    %h4.text-center
+      = f.text_field :title, style:"position: relative; left: 20%; width : 60%;"
```
Comment (alienware) [inline app/views/articles/_form.html.haml]: No inline styles, use CSS classes.

File: app/views/articles/index.html.haml [hunk 4]
```diff
@@ -1,18 +1,30 @@
-%h1 Listing articles
-= link_to 'New article', new_article_path
+- javascript_include_tag :defaults
```
Comment (alienware) [inline app/views/articles/index.html.haml:1]: delete

File: app/views/layouts/application.html.haml [hunk 5]
```diff
@@ -1,9 +1,10 @@
 %html
   %head
-    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
-    = javascript_include_tag 'application', 'data-turbolinks-track' => true
```
Comment (alienware) [inline app/views/layouts/application.html.haml:4]: Revert this change.

File: app/views/articles/_form.html.haml (hunk 3 above)
Comment (joney000) [inline app/views/articles/_form.html.haml]: removed all inline css

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (joney000) [inline app/assets/javascripts/application.js]: added new line

File: app/views/articles/index.html.haml (hunk 4 above)
Comment (joney000) [inline app/views/articles/index.html.haml:1]: deleted
